CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# dbt_google_ads v0.13.0
+
+## Schema Updates
+
+**2 total changes • 1 possible breaking change**
```

Comment:
Isn't adding a new column in both of these mean 2 potential breaking changes?

Reply:
I wouldn't say that a new column constitutes a breaking change necessarily. The real breaking change is that it may impact the grain of the end model